merchants/verkkokauppa/helpers.py

```diff
@@ -31,10 +31,14 @@ def get_formatted_reservation_time(reservation: Reservation) -> str:
     Weekday is localized based on user's preferred language, but rest
     of the format is always the same: ww dd.mm.yyyy hh:mm-hh:mm
     """
+    begin = reservation.begin.astimezone(get_default_timezone())
```

Comment:
Our default timezone is `Europe/Helsinki` is that also what is in the verkkokauppa? Or should it be converted to UTC?

Reply:
This is just a additional meta field we add to the order. We just send a string to Verkkokauppa and that string is displayed in the payment summary view. So we don't really send datetime to Verkkokauppa and it is not using this field internally.